src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -245,6 +254,26 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
         return oppsByRecurringDonationId;
     }
 
+    /**
+     * @description Handles payment counts
+     */
+    private void countOppPayments(List<Opportunity> opps) {
+        for (Opportunity opp : [
+            SELECT
```

Comment:
@seanpat09 I'm not sure why I didn't think of this before, but there is a rollup summary field on the Opportunity for `npe01__Number_of_Payments__c` that is a count of all payment records attached to the Opp. I think you can just use that rather than have to execute the subquery against the child object.

Reply:
haha nice catch! That's a lot more convenient, I will make the necessary updates